### PR TITLE
Awesome books day0

### PR DESCRIPTION
**Pull request by admirerbrown**
In this feature we:

- implemented day 2 project requirement
- fixed all linter errors in the project 
- ensured the project runs smoothly without errors.

### CODE REVIEW

File: newfeature.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+class BookScreen {
+  static addBooks() {
+    registeredBooks.innerHTML = '';
+    for (let i = 0; i < Books.length; i += 1) {
+      registeredBooks.innerHTML += `
+      <div class="title-and-author"> 
+        <p class="Title">"${Books[i].title}" by  ${Books[i].author}</p>
+        <button class="button" onclick="BookScreen.remove(${i})">remove</button>
+      </div>
```
Comment (OyePriscilla) [inline newfeature.js]: - [x] Please kindly add CSS styles to the application to make it match the given wireframe. 👍 

## Your Design 
book

File: newfeature.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
```
Comment (OyePriscilla) [inline newfeature.js]: - [x] Kindly please take note that according to best practices, we need to try our possible best not to disable any of the linters.

> - [ ] For line 1, kindly look through your codebase and remove any unused variable. 💁‍♂️ 
> - [ ] For line 2, you may have to define your variable before calling it. 💁‍♂️ 
> - [ ] For line 3, possibly place the two classes you have in your app in two different files; perhaps `book.js` and `bookscreen.js.` 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @admirerbrown, @annaeliana7, @Nuga0909 😃  , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Commit messages meaningful ✔️ 

## Required Changes ♻️

- [x] Kindly ensure to customize your [Readme](https://github.com/microverseinc/readme-template) file. This will give your repo a professional touch and be a guide to anyone that wants to access your project. Thanks. 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @admirerbrown @annaeliana7 @Nuga0909 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linters errors :heavy_check_mark: 
- App works as expected :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**